src/main/java/tech/pegasys/pantheon/plugin/services/metrics/Observation.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.plugin.services.metrics;
+
+import java.util.List;
+
+/** An Observation is registered to view Metrics data. */
+public interface Observation {
```

Comment:
We should keep this out of plugin APIs.

Reply:
Observations are redundant once the streams are removed.  I've deleted this class. 👍